libs/domain/cart/add/src/add.component.ts

```diff
@@ -65,6 +65,7 @@ export class CartAddComponent extends ProductMixin(
       ?disabled=${this.isInvalid || !this.$hasStock()}
       @click=${this.onSubmit}
       @mouseup=${this.onMouseUp}
+      replayable="click"
```

Comment:
We probably need to support multiple events, e.g. focus and hover. Array? Comma/space separated?

Reply:
trimming is missing